apps/storybook/src/stories/transactions/TransactionTable.tsx

```diff
@@ -0,0 +1,124 @@
+import { Badge } from "@/components/ui/badge";
+import { DataTable } from "@/components/ui/data-table";
+import { TransactionMeta } from "@/lib/domain/transaction/transaction";
+import { getShortAddress } from "@/utils/address";
+import { getShortHex } from "@/utils/hex";
+import { ColumnDef } from "@tanstack/react-table";
+import { ArrowUpDown, ExternalLink } from "lucide-react";
+import { formatEther } from "viem";
+
+export const txnTypeMap: Record<string, string> = {
+	contract_call: "Contract Call",
```

Comment:
duplicated with `findDisplayedTxType` in `asTransactionMeta`
apps/storybook/src/lib/blockscout/api.ts

Reply:
hmm doesnt seem so, `findDisplayedTxType` is re-aliasing of the naming, while `txnTypeMap` kinda converts the `snake_case` name to a plain language.


```ts
export const findDisplayedTxType = (transaction_types: any[]): string => {
	if (transaction_types.includes("contract_call")) {
		return "contract_call";
	}
	if (transaction_types.includes("coin_transfer")) {
		return "coin_transfer";
	}
	return "native_transfer";
};
```
```ts
export const txnTypeMap: Record<string, string> = {
  contract_call: "Contract Call",
  native_transfer: "Native Token Transfer",
  token_transfer: "Token Transfer",
  coin_transfer: "Token Transfer",
};
```

Or would you prefer to  render the snake_case in the table?

Reply:
oh my bad you're right, I quoted the wrong code
haven't even merge that
Let's use yours first I can follow up